src/test/java/io/confluent/idesidecar/restapi/util/cpdemo/Constants.java

```diff
@@ -0,0 +1,30 @@
+package io.confluent.idesidecar.restapi.util.cpdemo;
+
+public class Constants {
+  private Constants() {
+  }
+
+  static String SSL_CIPHER_SUITES = "TLS_AES_256_GCM_SHA384,TLS_CHACHA20_POLY1305_SHA256,TLS_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256";
+
+  public static final String BROKER_JAAS_CONTENTS = """
+      Client {
+             org.apache.zookeeper.server.auth.DigestLoginModule required
+             username="kafka"
+             password="kafkasecret";
+      };
+
+      internalhost.KafkaServer {
+           org.apache.kafka.common.security.plain.PlainLoginModule required
+           username="admin"
+           password="admin-secret"
+           user_admin="admin-secret"
+           user_mds="mds-secret";
+      };
+
+      tokenhost.KafkaServer {
+           org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required
+           publicKeyPath="/tmp/conf/public.pem";
+      };
```

Comment:
@rhauch I had to add this for the additional hosts to work. Else, the cp-server container shuts down with "internalhost.KafkaServer not defined in JAAS config"

I don't understand why this wasn't needed for the other existing listeners.

Reply:
I'm not really sure why cp-server behaves this way, and I couldn't find anything about `not defined in JAAS config` or `internalhost.KafkaServer`. Maybe the token mechanism specified elsewhere is checking for this in the config? That might explain why it wasn't needed for other listener configs.